### PR TITLE
Docs: A few README updates

### PR DESCRIPTION
**Pull request by jhaaaa**
- Update "General Availability" status to "Production" status according to revised status names
- Remove content about conversation IDs
- Update links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-To learn more about XMTP and get answers to frequently asked questions, see [FAQ about XMTP](https://xmtp.org/docs/dev-concepts/faq).
+To learn more about XMTP and get answers to frequently asked questions, see the [XMTP documentation](https://xmtp.org/docs).
 
 ![x-red-sm](https://user-images.githubusercontent.com/510695/163488403-1fb37e86-c673-4b48-954e-8460ae4d4b05.png)
 
-## Example app
+## Quickstart app built wtih `xmtp-flutter`
 
-For a basic demonstration of the core concepts and capabilities of the `xmtp-flutter` client SDK, see the [Example app project](https://github.com/xmtp/xmtp-flutter/tree/main/example).
+Use the [XMTP Flutter quickstart app](https://github.com/xmtp/xmtp-quickstart-react) as a tool to start building an app with XMTP. This basic messaging app has an intentionally unopinionated UI to help make it easier for you to build with.
```
Comment (dmccartney) [inline README.md:17]: I don't think this is right? (looks like copy-paste from react doc but flutter should still point at the flutter example app).

**Review by dmccartney — APPROVED** [review]
I think one note/link is wrong.
But after that, LGTM

File: README.md (hunk 1 above)
Comment (jhaaaa) [inline README.md:17]:
```suggestion
Use the [XMTP Flutter quickstart app](https://github.com/xmtp/xmtp-flutter/tree/main/example) as a tool to start building an app with XMTP. This basic messaging app has an intentionally unopinionated UI to help make it easier for you to build with.
```

Comment (jhaaaa) [reply README.md]: My apologies - thank you for catching this, @dmccartney - fix committed!

**Pull request merged**